package.json

```diff
@@ -16,8 +17,12 @@
   "author": "",
   "license": "BSD",
   "dependencies": {
+    "@aws-sdk/client-s3": "^3.208.0",
+    "@aws-sdk/url-parser": "^3.208.0",
     "@types/bcrypt": "^5.0.0",
-    "@types/sha.js": "^2.4.0"
+    "@types/sha.js": "^2.4.0",
+    "lodash": "^4.17.21",
+    "uuidv4": "^6.2.13"
```

Comment:
This doesn't look to be used but if you do need uuids then the `uuid` module makes more sense as it's a much wider used/more maintained project: https://www.npmjs.com/package/uuid

Reply:
nice, this was moved to the S3-storage-adapter package, will check it there